### DISCUSSION TITLE
Ironwail remastered issues

### DISCUSSION THREAD

**Discussion by huahuahuahuah** [andrei-drexler/ironwail#291] [Q&A]
When I unzip ironwail on my desktop then launch it it has some features from the remaster, like better lighting and sound, music, but doesnt let me launch eoe but when i unzip into a copy of my steam folder it does not give me features of the remaster but i can start eoe fine is there a way i can play custom maps and also get the better graphics?

**Comment by ODDTIMEBALL** [comment]
Hey I actually figured this one out. while I don't know why it defaults to original when selecting it from the steam folder/button, you can get custom maps to load with the remastered version by putting a copy of the map in the rerelease folder on steam. I was having the same issue as you until I though to do that.
